saltstack/base/salt/prerequisites/usr/bin/ccmv2-helper.sh

```diff
@@ -12,11 +12,11 @@ setup_ccmv2() {
 
   mkdir -p /etc/ccmv2
 
-  IV=436c6f7564657261436c6f7564657261
+  LEGACY_IV=436c6f7564657261436c6f7564657261
   AGENT_KEY_PATH=/etc/ccmv2/ccmv2-key.enc
 
   CCM_V2_AGENT_KEY_HEX=$(xxd -pu -l 16 <<< $CCM_V2_AGENT_KEY_ID)
-  echo ${CCM_V2_AGENT_ENCIPHERED_KEY} | openssl enc -aes-128-cbc -d -A -a -K ${CCM_V2_AGENT_KEY_HEX} -iv ${IV} > ${AGENT_KEY_PATH}
+  echo ${CCM_V2_AGENT_ENCIPHERED_KEY} | openssl enc -aes-128-cbc -d -A -a -K ${CCM_V2_AGENT_KEY_HEX} -iv ${LEGACY_IV} > ${AGENT_KEY_PATH}
```

Comment:
There should be an equivalent of this when we use aes-256 for encryption as well.

Reply:
well, it works (tested both cases)

Reply:
Ok. As clarified by Denes over slack, AGENT_KEY_PATH is required in case of mTLS only.